### DISCUSSION TITLE
Collective Name for the OctoposPrime Community

### DISCUSSION THREAD

**Discussion by Sddilora** [octoposprime/op-docs#22] [Ideas]
Hello OctoposPrime friends!

We are currently in search of a name that reflects the spirit of our project and the identity of our community. As you may have noticed, we have chosen our octopuses as the mascots of the project, and it's time to give these adorable creatures a name!

Let's draw inspiration from a few examples: As you may know, the community of the Go programming language is called "Gopher." This name not only reflects their dynamic and helpful nature but also conveys a sense of unity and solidarity. Similarly, the Ruby community is referred to as "Rubyists," while the Python community is known as "Pythonistas." These names not only denote users of a language but also make you feel like a member of a family.

Now it's our turn to choose a name for our OctoposPrime octopus community! Perhaps drawing inspiration from the depths of the sea, we could give our octopuses a name like "Deepers." Or we could opt for a name like "Flexopus" to highlight their flexible and intelligent nature.

This GitHub discussion is a call to hear your ideas and suggestions. Please join in and help us find a name that will represent and unite our community!

Thank you!

**Comment by betulbodur00** [comment]
I've got an idea.

Octonotri (octonotris as in plural)

Based on numbers.

Octo, because octopuses have 8 legs/arms, and it resembles the word octopus.

No is based on nine, since octopuses have 9 brains. (From the Latino word novem (nine))

And tri, for 3 since octopuses has 3 hearts.

So I've merged those numbers to make a (completely new) word. It's just an idea. But it may be a little hard to pronounce.

**Reply by husamettinarabaci** [reply]
This idea is amazing..
We are Octonotris..
